hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -268,7 +269,12 @@
               for (int j = 0; j < numMetaReplicas; j++) {
                 RegionInfo meta = RegionReplicaUtil.getRegionInfoForReplica(
                         RegionInfoBuilder.FIRST_META_REGIONINFO, j);
-                ServerName metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+                ServerName metaLocation = null;
+                try {
+                  metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+                } catch (NotAllMetaRegionsOnlineException e) {
+                  //Should ignore this Exception for we don't need to display rit meta region info in UI
+                }
```

Comment:
In this case, the user is looking at the meta table specifically. Maybe the below `if (metaLocation != null) {...` needs an `else` block that prints a message stating that some region location(s) could not be identified?

Reply:
> In this case, the user is looking at the meta table specifically. Maybe the below `if (metaLocation != null) {...` needs an `else` block that prints a message stating that some region location(s) could not be identified?

Since the user may learn the meta region is in transition state from the transition list in Master status UI， Should we still show any details here?

After I check the current implementation it shows that If a region of user-level table is in transition, it does not be displayed in its table details page. So I think the meta region should be the same.

Reply:
Oh, this is the current behavior? In the table detailed page, usually we will sort the regions by start key, and if we miss some regions because they are in transition, there will be holes on the page? Seems a bit strange.

Reply:
My mistake! After checking again, I found that the detailed page does display info of region in transaction. The new commit  will submit immediately

Reply:
Now,  region in transition here would be displayed and all its value is 0 or N/A(region location would be set as an empty string)